### PR TITLE
Implements configurable models

### PR DESCRIPTION
**Pull request by nicolaibaaring**
This PR adds configurable models and addresses issue #39. 
The configuration has been implemented in the same exact style as in the other Laravel Cashier packages.

I doesn't add any new behavior and existing tests are passing. Tests of the configurability have been added.

### CODE REVIEW

File: src/CashierServiceProvider.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace Laravel\Cashier;
 
 use Illuminate\Support\ServiceProvider;
+use Laravel\Cashier\Cashier;
```
Comment (sandervanhooft) [inline src/CashierServiceProvider.php:6]: No need

**Review by sandervanhooft — CHANGES_REQUESTED** [review]
Super nice! 🤩 

To complete this PR, could you:

1. Remove the now obsolete imports. For example `use Laravel\Cashier\Order\OrderItem;` is now obsolete.
2. Add some documentation (we may need a new section for this)

**Comment by sandervanhooft** [comment]
Also, there's some tests failing. Probably an easy fix.

File: src/CashierServiceProvider.php (hunk 1 above)
Comment (nicolaibaaring) [reply src/CashierServiceProvider.php:6]: Yes, I left them there for this exact request to come up. 😂 I'll remove them.

Comment (nicolaibaaring) [reply src/CashierServiceProvider.php:6]: Or do you actually want explicit imports in the same namespace removed? Not only obsolete ones?

Comment (nicolaibaaring) [reply src/CashierServiceProvider.php:6]: Import statements are removed and submitted. Documentation has been added in a separate section.

**Comment by nicolaibaaring** [comment]
The code that made tests fail has been fixed.

**Comment by sandervanhooft** [comment]
Thanks @nicolaibaaring ! 

I noticed a large number of lingering imports even after this commit: https://github.com/mollie/laravel-cashier-mollie/pull/48/commits/d6e48d823a222078f1ff30f88b16ebc236daa504

Can you double check this?

**Comment by nicolaibaaring** [comment]
@sandervanhooft, did you verify, if this is completed, and is there any chance that this can be merged?

**Review by sandervanhooft — APPROVED** [review]

**Comment by sandervanhooft** [comment]
All well!

Just curious: what sparked this? What default behaviour were you missing?

**Comment by nicolaibaaring** [comment]
@sandervanhooft I never originally saw that question, but now three years later, I was pinged about it. :joy: Sorry for never answering back then.

The answer is that I build modular code and wanted to keep this package in a Billing module. That includes the database tables, so I needed to keep them together in the database. That is done by prefixing the table names with the module prefix, and making the models configurable made that possible.

**Comment by sandervanhooft** [comment]
Makes sense! It's still working like a charm, thanks!

**Pull request merged**